### PR TITLE
PUBDEV-6101 fix StackedEnsembleMojoSubModel to be serializable

### PR DESCRIPTION
**Pull request by maurever**
The `StackedEnsembleMojoModel` has a static inner class `StackedEnsembleMojoSubModel` which is not serializable. This PR fixes the `StackedEnsembleMojoSubModel` implements the `Serializable` interface. 

JIRA: https://0xdata.atlassian.net/browse/PUBDEV-6101

### CODE REVIEW

**Comment by maurever** [comment]
I am just curious, but is it necessary to have the static inner class there? What is the reason? @Pscheidl

**Comment by michalkurka** [comment]
@maurever please also add a test of the serialization

**Comment by maurever** [comment]
> @maurever please also add a test of the serialization

Of course... Done.

I checked others MOJO models tests, and anyone has a serializable test case...

File: h2o-genmodel/src/test/java/hex/genmodel/algos/ensemble/StackedEnsembleSerializableMojoTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                in.close();
+            } catch (IOException e) {
+                Assert.fail(e.toString());
+            }
+        }
+    }
+
+    private byte[] serialize(final Serializable input) {
+        final ByteArrayOutputStream out = new ByteArrayOutputStream();
+        try {
```
Comment (Pscheidl) [inline h2o-genmodel/src/test/java/hex/genmodel/algos/ensemble/StackedEnsembleSerializableMojoTest.java]: Please use try-with-resources block.
```java
try(final ObjectOutputStream objectOutput = new ObjectOutputStream(out)){
....
}
```

**Pull request merged**